man/ukify.xml

```diff
@@ -25,16 +25,25 @@
       <command>/usr/lib/systemd/ukify</command>
       <arg choice="opt" rep="repeat">OPTIONS</arg>
       <arg choice="plain">build</arg>
+    </cmdsynopsis>
+
+    <cmdsynopsis>
+      <command>/usr/lib/systemd/ukify</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
       <arg choice="plain">genkey</arg>
     </cmdsynopsis>
+
+    <cmdsynopsis>
+      <command>/usr/lib/systemd/ukify</command>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+      <arg choice="plain">inspect</arg>
+      <arg choice="plain" rep="repeat">FILE</arg>
+    </cmdsynopsis>
   </refsynopsisdiv>
 
   <refsect1>
     <title>Description</title>
 
-    <para>Note: this command is experimental for now. While it is intended to become a regular component of
```

Comment:
Do we want to move it to `/usr/bin/` with this?

Reply:
Oh, we do.